src/main/java/adris/altoclef/tasksystem/Task.java

```diff
@@ -182,4 +182,8 @@ private boolean canBeInterrupted(AltoClef mod, Task subTask, Task toInterruptWit
             return true;
         }));
     }
+
+    public String getItemName() {
```

Comment:
This should be a part of `ResourceTask`, not `Task.java`.
(Not all tasks are item tasks. Movement, Construction, Idling, etc.)